native/core/src/spaces.rs

```diff
@@ -223,7 +240,7 @@ impl CoreClient {
                 // local uri
                 let path = PathBuf::from(avatar_uri);
                 let guess = mime_guess::from_path(path.clone());
-                let content_type = guess.first().expect("don’t know mime type");
+                let content_type = guess.first().expect("don't know mime type");
```

Comment:
Frankly, I used prime instead of single quote, because single quote is default of flutter side and double quote is default of rust side.

Reply:
this was just the editor auto-setting it. If we want to ensure a specific style we should put that into an editorconfig or something.